packages/engine/Source/Core/ITwin.js

```diff
@@ -0,0 +1,306 @@
+import defined from "./defined.js";
+import DeveloperError from "./DeveloperError.js";
+import Resource from "./Resource.js";
+import RuntimeError from "./RuntimeError.js";
+
+/**
+ * @enum {string}
+ */
+const ExportStatus = Object.freeze({
+  NotStarted: "NotStarted",
+  InProgress: "InProgress",
+  Complete: "Complete",
+  Invalid: "Invalid",
+});
+
+/**
+ * Type of an export currently, only GLTF and 3DFT are documented
+ * The CESIUM option is what we were told to use with Sandcastle
+ * I've also seen the IMODEL one but don't know where it's from
+ * @enum {string}
+ */
+const ExportType = Object.freeze({
+  "3DFT": "3DFT",
+  GLFT: "GLTF",
+  IMODEL: "IMODEL",
+  CESIUM: "CESIUM",
+});
```

Comment:
We're currently working on the `mesh-export` API, and we changed this quite a bit. I updated the documentation on Monday, but it will take some time to be deployed.
3DFT and GLTF are now deprecated, IMODEL is for internal use only (until we deprecate it) and now you can use "3DTILES", which is what I would recommend to use.